index.html

```diff
@@ -26,30 +26,77 @@ <h2 id="project_tagline"></h2>
 <!-- MAIN CONTENT -->
 <div id="main_content_wrap" class="outer">
     <section id="main_content" class="inner">
-        <h2>
-            <a id="eiffel-remrem-protocol-interface" class="anchor" href="#eiffel-remrem-protocol-interface" aria-hidden="true"><span
-                    aria-hidden="true" class="octicon octicon-link"></span></a>Eiffel REMReM Protocol Interface
-        </h2>
         <h2>Introduction</h2>
         <p>Eiffel REMReM Protocol Interface project contains java interfaces that are implemented by Eiffel protocol</p>
         <p>Implementing an interface allows a class to become more formal about the behavior it promises to provide. Interfaces form a contract
             between the class and the outside world, and this contract is enforced at build time by the java compiler.</p>
         <h2>Installation</h2>
         <p>This project does not need any installation since it is an interface, which is implemented by other projects.</p>
         <p>This project is build by gradle, by executing following commands:</p>
-        <p>./gradlew build</p>
+        <pre>./gradlew build</pre>
         <p>The produced jar file will be stored in /eiffel-remrem-protocol-interface/build/libs/</p>
         <h2>Configuration</h2>
         <p>This project does not need any configuration since it is an interface, which is implemented by other projects.</p>
         <h2>Compatibility</h2>
         <p>All message services need to implement interface provided by this project to communicate properly.</p>
         <h2>Usage</h2>
-        <p>To create new version of interface you need to modify the interface, change the java api docs for methods to represent new version of
-            interface and build the jar using gladle. Then changes need to be done in
-            <a href="https://github.com/Ericsson/eiffel-remrem-semantics">eiffel-remrem-semantics</a>,
-            <a href="https://github.com/Ericsson/eiffel-remrem-generate">eiffel-remrem-generate</a>,
-            <a href="https://github.com/Ericsson/eiffel-remrem-publish">eiffel-remrem-publish</a>
-        </p>
+        <p>An interface in the Java programming language is an abstract type that is used to specify a behavior that classes must implement.
```

Comment:
The text is kind of written as a school book for Java and Java Interfaces.   The focus should be on a programmer that wants to implement a new protocol...  I think like describing in high level the different service methods should provide and what they do. On a more higher level. If they later want to know more details they should look at JavaDoc